origami.json

```diff
@@ -5,11 +5,24 @@
     "support": "next.team@ft.com",
     "supportStatus": "experimental",
     "runServer": "true",
+    "demosDefaults": {
+        "sass": "demos/src/demo.scss",
+        "js": "demos/src/demo.js",
+        "documentClasses": "",
+        "dependencies": "o-fonts@^2.0.0,o-grid@^4.0.0"
+    },
     "demos": [
         {
-            "path": "/demos/light-signup-demo.html",
-            "expanded": true,
-            "description": ""
+            "name": "promo-image",
+            "description": "Light signup form with promo image",
+            "template": "demos/src/standard.html",
+            "path": "/demos/promo-image.html"
```

Comment:
I looked at the makefile this morning with much fresher eyes, it seems that `make demo` also updates the `origami.json`.

``` make
demo:
	obt demo --runServer --updateorigami
```

If you try removing the paths and running the following:

```
obt demo --runServer
```
That will not update `origami.json` and therefore no new file changes for git to pick up on. I would say you can remove the paths and use templates exclusively. Also remove the makefile. Origami doesn't really use makefiles, but rather with OBT itself.